core/src/main/scala/com/databricks/labs/remorph/generators/sql/LogicalPlanGenerator.scala

```diff
@@ -97,38 +100,37 @@ class LogicalPlanGenerator(val expr: ExpressionGenerator, val explicitDistinct:
     s"(${generate(ctx, setOp.left)}) $op$duplicates (${generate(ctx, setOp.right)})"
   }
 
-  private def generateMerge(ctx: GeneratorContext, mergeIntoTable: ir.MergeIntoTable) = {
+  private def update(ctx: GeneratorContext, update: UpdateTable): String = {
+    val target = generate(ctx, update.target)
+    val source = update.source.map(generate(ctx, _)).getOrElse("")
+    val set = update.set.map(expr.generate(ctx, _)).mkString(", ")
+    val where = update.where.map(expr.generate(ctx, _)).getOrElse("")
+    s"UPDATE $target SET $set $source WHERE $where"
```

Comment:
Is this correct? [the documentation](https://docs.databricks.com/en/sql/language-manual/delta-update.html) doesn't seem to specify anything appearing between the assignments (introduced by `SET`) and the predicates (introduced by `WHERE`). 

Also, this code would ouptut `WHERE` even if no predicate is specified.

Reply:
thanks!